test/ssl-tests/02-protocol-version.conf

```diff
@@ -18787,7 +18787,7 @@ PrivateKey = ${ENV::TEST_CERTS_DIR}/serverkey.pem
 
 [676-ciphersuite-sanity-check-client-client]
 CipherString = AES128-SHA
-Ciphersuites = 
+Ciphersuites =
```

Comment:
This is a generated file, and changing this requires changing `test/ssl_test.tmpl`.  I suggest undoing this change.

Reply:
I will undo these.